lms/static/js/groups/views/cohort_editor.js

```diff
@@ -57,9 +71,18 @@ var edx = edx || {};
             cohortFormView.saveForm()
                 .done(function() {
                     cohortFormView.showMessage(gettext('Saved cohort'));
```

Comment:
Note that the page reload (which we want to get rid of) means that the "Saved cohort" message is never even visible now.